libs/utils/src/string.ts

```diff
@@ -0,0 +1,3 @@
+export const capitalize = (s: string) => {
+  return s[0].toUpperCase() + s.slice(1).toLowerCase();
+};
```

Comment:
https://lodash.com/docs/4.17.15#capitalize

Reply:
I understand, but I actively want to _discourage_ new dependencies unless absolutely needed. I mean, we could pull in `lodash.capitalize`, but why? It's super simple to write.

Reply:
Thinking -- write less, do more

Reply:
converted to lodash-es